web/modules/custom/customnewblock/customnewblock.install

```diff
@@ -0,0 +1,78 @@
+<?php
+
+use Drupal\block\Entity\Block;
+use Drupal\Core\Config\FileStorage;
+/**
+ * Some information about the update.
```

Comment:
@anitha-innoraft - Function comments should specify the purpose of the function.

Reply:
Done